test/tools/fuzzer_common.h

```diff
@@ -24,7 +24,8 @@
 struct FuzzerUtil
 {
 	static void runCompiler(std::string const& _input, bool _quiet);
-	static void testCompiler(std::string const& _input, bool _optimize, bool _quiet);
+	static void testCompilerJsonInterface(std::string const& _input, bool _optimize, bool _quiet);
 	static void testConstantOptimizer(std::string const& _input, bool _quiet);
 	static void testStandardCompiler(std::string const& _input, bool _quiet);
+	static void testCompiler(std::string const& _input, bool _optimize);
```

Comment:
Is this function still used?

Reply:
Yes, it is used here

https://github.com/ethereum/solidity/blob/435c9dae718eeef645edb715293c4c687240ee5e/test/tools/afl_fuzzer.cpp#L127

Reply:
Oops, ignore my previous comment.

`testCompiler` has been renamed `testCompilerJsonInterface`. This is now used exclusively in the afl-fuzzer harness. This API does not leak exceptions and has been retained for backwards compatibility with solfuzzer.

The new harness is called `testCompiler`. This is now used in test files inside ossfuzz sub-directory. This harness leaks exceptions.